.travis.yml

```diff
@@ -108,7 +114,7 @@ jobs:
 
     - <<: *base-test
       stage: test
-      name: "watch, storage, app, project, push, devfile catalog, devfile create, devfile push, devfile delete command integration tests"
```

Comment:
Add this line back. May be in the rebase somehow you missed it

Reply:
done